### DISCUSSION TITLE
Help in moving/archiving LTSP5 content

### DISCUSSION THREAD

**Discussion by alkisg** [ltsp/ltsp#208] [Q&A]
TL;DR: Any users that care about LTSP5 content should help in moving/archiving it or it will be lost.

Longer version; we're preparing the following changes:

- ltsp.github.io will move to the ltsp.org domain (the storage will still be github)
- irclogs.ltsp.org will be preserved in some form

The following will be lost, unless users offer to move the useful content to https://github.com/ltsp/ltsp5/wiki:

- http://wiki.ltsp.org content will be lost upon domain move
- http://www.ltsp.org will be lost too, including success stories. If a web developer wants to move these, it'd be awesome, but I guess it's a bigger task.
- https://help.ubuntu.com/community/UbuntuLTSP will be deleted in April when Ubuntu 20.04 is released with the new LTSP, as the content there is horribly unmaintained and out of date.

**Comment by CharlesMAtkinson** [comment]
I could work on moving http://wiki.ltsp.org content but am not familiar with it (or LTSP yet) so do not know which content is worth moving.

**Comment by alkisg** [comment]
Thank you Charles,

ok, so anyone interested in any wiki.ltsp.org pages, mention them here, so that they can be preserved. I'll also send a message to the ltsp-discuss mailing list about it.
